src/main/java/org/intellij/sdk/codesync/files/ConfigFile.java

```diff
@@ -32,9 +31,7 @@ public ConfigFile(String filePath) throws InvalidConfigFileError {
         try {
             this.contentsMap = this.readYml();
         } catch (InvalidYmlFileError e) {
-            CodeSyncYmlFile.removeFileContents(this.getYmlFile());
-            StateUtils.reloadState(StateUtils.getGlobalState().project);
```

Comment:
We will only be logging the error now in case of yaml error? was there any problem with reset logic?